source/_integrations/homekit_controller.markdown

```diff
@@ -137,9 +137,10 @@ This section describes how to add it via Home Assistant's preferred {% term Thre
 - **Thread network**: In order to use HomeKit over Thread, you need a working border router.
   - Make sure your Home Assistant device is on the same network (LAN) as the border router.
   - Make sure the {% term Thread %} network you'd like to use is known by Home Assistant and marked as **Preferred network** in the {% term Thread %} configuration.
-  - If you have a Home Assistant Yellow or Connect ZBT-1, you can enable Thread to set up an Open Thread border router and with that a {% term Thread %} network.
+  - If you have a Home Assistant Yellow, or a Connect ZBT-1 or ZBT-2, you can enable Thread to set up an Open Thread border router and with that a {% term Thread %} network.
```

Comment:
nit: double "or". Maybe this?
```suggestion
  - If you have a Home Assistant Yellow, Connect ZBT-1, or ZBT-2, you can enable Thread to set up an Open Thread border router and with that a {% term Thread %} network.
```

or to mention "Connect" twice:
```suggestion
  - If you have a Home Assistant Yellow, Connect ZBT-1, or Connect ZBT-2, you can enable Thread to set up an Open Thread border router and with that a {% term Thread %} network.
```

... or with the non-breaking spaces, similar to the Thread integration docs:

```suggestion
  - If you have a Home Assistant Yellow, Connect&nbsp;ZBT-1, or Connect&nbsp;ZBT-2, you can enable Thread to set up an Open Thread border router and with that a {% term Thread %} network.
```